packages/puppeteer-core/src/cdp/Connection.ts

```diff
@@ -83,16 +83,23 @@ export class Connection extends EventEmitter<CDPSessionEvents> {
   /**
    * @internal
    */
-  get _sessions(): Map<string, CDPSession> {
+  get _sessions(): Map<string, CdpCDPSession> {
     return this.#sessions;
   }
 
+  /**
+   * @internal
+   */
+  _session(sessionId: string): CdpCDPSession | null {
```

Comment:
This was introduced but not used, please remove.

Reply:
It is used in https://github.com/puppeteer/puppeteer/pull/13620/files#diff-b688d84d1dee4967409766c8b2061c79c314e6e9cb548683adf7d14655b07b57L321

Reply:
Sorry VsCode somehow didn't find the reference.

Reply:
I think I got what you've meant, removed